pkg/cmd/workspace/code.go

```diff
@@ -191,6 +195,9 @@ func init() {
 	}
 	ideListStr := strings.Join(ids, ", ")
 	CodeCmd.Flags().StringVarP(&ideFlag, "ide", "i", "", fmt.Sprintf("Specify the IDE (%s)", ideListStr))
+
+	CodeCmd.Flags().BoolP("yes", "y", false, "Automatically confirm any prompts")
```

Comment:
Please use `BoolVarP` so it's aligned with the rest of the codebase. (You have the `autoStartFlag` as an example)